### PR TITLE
Md files checking and updating

### LINKED ISSUE #11868: Examples README.md files needs checking and updating
Some of the `README.md` files in the `examples` directory and its subdirectories are out of date, so need checking and updating.

As an example, `plotting/README.md` contains
```
There are three subdirectories:

* `file` examples that output to static HTML files

* `notebook` examples that display inline in [Jupyter](http://jupyter.org) notebooks

* `server` examples that send output to a Bokeh server, using the python client library for the server
```
but there are no `notebook` and `server` directories.

### PR DESCRIPTION
**Pull request by girolamodaschio**
- [x ] issues: fixes #11868
- [ ] tests added / passed
- [ ] release document entry (if new feature or API change)

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -21,6 +25,10 @@ some  particular aspect of Bokeh capability (such as [linking and
 brushing](http://www.infovis-wiki.net/index.php?title=Linking_and_Brushing)),
 or walk through a particular example in additional detail.
 
+### [`Integration`](integration/)
+
+This directory contains code useful with Bokeh's integrations.
```
Comment (bryevdv) [inline examples/README.md:30]: The examples in the Integration subdirectory are all specifically crafted for image-diff integration tests. They are in this location as a technical compromise, but mostly are probably are not interesting/useful to users. I would suggest simply omitting mention of this directory of the README altogether

File: examples/plotting/README.md [hunk 2]
```diff
@@ -12,10 +14,6 @@ associated with your data. A simple, but typical complete example of this API lo
 
     show(p)
 
-There are three subdirectories:
-
-* `file` examples that output to static HTML files
-
-* `notebook` examples that display inline in [Jupyter](http://jupyter.org) notebooks
+## Structure
```
Comment (bryevdv) [inline examples/plotting/README.md:17]: I don't think an entire new subsection is needed just for one sentence, I would delete this and just leave the sentence below on its own.

**Comment by bryevdv** [comment]
Thanks @girolamodaschio I have left a few comments above

File: examples/README.md (hunk 1 above)
Comment (mattpap) [reply examples/README.md:30]: Exactly; integration examples are on they way out, so there is no point in mentioning them anymore.

Comment (girolamodaschio) [reply examples/README.md:30]: Yess, thank you for the suggestion, I'll update the file

**Comment by girolamodaschio** [comment]
Thanks @bryevdv  for the suggestion, I've updated the branch.

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @girolamodaschio !

**Pull request merged**